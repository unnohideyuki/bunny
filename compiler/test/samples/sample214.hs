main = do print $ subtract 3 5
          print $ subtract 3 (5 :: Int)
          print $ subtract (3 :: Double) 5
          print $ even 5
          print $ odd 3
