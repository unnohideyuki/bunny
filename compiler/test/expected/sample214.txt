2
2
2.0
False
True
